### DISCUSSION TITLE
upgrading from 13 to tensor beta

### DISCUSSION THREAD

**Discussion by Sceptersax** [blakeblackshear/frigate#9694] [General]
I know this is a simple question and I tried to find the answer but i'm failing to do so, so I'll have to ask. I had a working version of 12 working great with Tensorrt but I need to be able to incorporate the PTZ UI controls. I updated to 13 by changing the image name to beta2 but i didn't see the UI. What is the latest image that allows for this new feature and what's the best way to upgrade? Is it simply by changing the image name in the docker compose file to the beta tensorrt image?

So i guess the question is 2 fold:

1. What is the tensorrt image that has the PTZ control feature
2. How do I update my frigate version 12 to that image?

Thanks

**Comment by NickM-27** [comment]
1. stable-tensorrt
2. Update the compose file. docker compose pull then docker compose up -d

**Reply by Sceptersax** [reply]
Awesome but which image? The latest 13? or is it in beta?

**Reply by NickM-27** [reply]
the image is stable-tensorrt like I said

**Reply by NickM-27** [reply]
https://docs.frigate.video/frigate/installation#docker

**Reply by Sceptersax** [reply]
Ok great i see the doc but are you saying the PTZ UI controls are part of the latest non beta? Sorry for the inconvenience.

**Reply by NickM-27** [reply]
Yes, like it says in the 0.13 release notes

**Reply by Sceptersax** [reply]
Ok thanks
